README.md

```diff
@@ -53,6 +53,11 @@ packmap -p ../my-package/package.json
 
 # specify path to import-map which overrides generated import-map
 packmap --override-map ./override-import-map.json
+
+# specify current working directory to base relative urls on.
+# In this example, the main package.json will be loaded from subdir, and the
+# outputted dist director will also be created inside of subdir.
+packmap --cwd ./subdir
```

Comment:
So this is equivalent to `cd subdir; packmap; cd ..` ?

Reply:
Yep!